docs/source/markdown/podman-images.1.md

```diff
@@ -15,11 +15,11 @@ Displays locally stored images, their names, and their IDs.
 
 ## OPTIONS
 
-#### **--all**, **-a**
+#### **--all**, **-a**=*true|false*
```

Comment:
I don't think we should add this to boolean options.
At least that is what is written in [MANPAGE_SYNTAX](https://github.com/containers/podman/blob/main/docs/MANPAGE_SYNTAX.md)
> If the arguments behind the OPTION are boolean, it is not shown behind the OPTION itself. The default boolean argument is shown in the same way normal default arguments are displayed.
Example: The default is false.

Reply:
I don't know, I am fine with removing them, but I was striving for consistency.